src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +26,23 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        InputStream inputStream = Files.newInputStream(Paths.get(fileName));
+        int bytesOfFile;
+        StringBuilder onlyLetters = new StringBuilder();
+        do {
+            bytesOfFile = inputStream.read();
+            if ((bytesOfFile > 64 && bytesOfFile < 91) || (bytesOfFile > 96 && bytesOfFile < 123)) {
```

Comment:
@kenu21 what means 64? 91? 96? 123? Why this numbers?

Reply:
@kenu21 you may do this more simpler

Reply:
I created final int. Sofia said me it today - isLetter(). Can I do it in my way?